src/Utilities/hooks/useTableTools/Components/FlexibleFlex.js

```diff
@@ -2,15 +2,24 @@ import React from 'react';
 import { Flex, FlexItem } from '@patternfly/react-core';
 import propTypes from 'prop-types';
 
-const renderFlexItem = (display) => {
-  return <FlexItem>{display}</FlexItem>;
+const renderFlexItem = (content) => {
+  let flexItem;
+  content?.classname
+    ? (flexItem = (
+        <FlexItem className={content.classname}>{content.display}</FlexItem>
```

Comment:
Is there any purpose of using `display` key for content object? If it is designed for only this PR or a specific component, we can make this key more flexible with more dynamic key so that it can be used in the future as well (e.g `text` or `children`).
`<FlexItem className={content.classname}>{content.text}</FlexItem>`

This is a minor suggestion, you can ignore it if you do not think it does not make sense.

Reply:
Is the new change what you were referring to?